warehouse/warehouse.go

```diff
@@ -887,13 +887,14 @@ func Start(ctx context.Context, app app.App) error {
 			return monitorDestRouters(gCtx)
 		}))
 
-		archiver := &archive.Archiver{
-			DB:          dbHandle,
-			Stats:       stats.Default,
-			Logger:      pkgLogger.Child("archiver"),
-			FileManager: filemanager.New,
-			Multitenant: tenantManager,
-		}
+		archiver := archive.New(
+			config.Default,
+			logger.NOP,
```

Comment:
Why a `NOP` logger? :thinking:

Reply:
Thanks for pointing out. It happened by mistake.